### DISCUSSION TITLE
/search CSRF token mismatch after migrating to live environment

### DISCUSSION THREAD

**Discussion by cvassios-eibach** [Laravel-Backpack/community-forum#1006] [Q&A (Help)] [answered]
Hello,

I recently got my Backpack admin ready to go, but on my live environment, every list page I got to shows a 419 error message that "CSRF token mismatch." and I'm struggling to figure out why. Any ideas?

**Comment by pxpm** [comment, marked as answer]
Hey @cvassios-eibach 

That error means that the assets are not properly loaded. We add the CSRF token in `common.js` if you look at your Developer Console in network tab, you should see those files with error.

Is your APP_URL correctly set in your live environment ? 

does `php artisan basset:check` return all good ? 

Do you run `php artisan optimize:clear` before the `php artisan optimize` ? 

Does `php artisan basset:clear && php artisan basset:cache` create the expected files in the `storage/app/public/basset` folder ? 

Did you create the symlink with `php artisan storage:link` ? 

Let me know the results of those. 

Cheers

**Reply by cvassios-eibach** [reply]
DOH! My symlink was removed, that did it. Thank you!
